nupic/research/fast_temporal_memory.py

```diff
@@ -182,9 +182,9 @@ def cellsForColumn(self, column):
     """
     self._validateColumn(column)
 
-    start = self.cellsPerColumn * column
-    end = start + self.cellsPerColumn
-    return set([ConnectionsCell(idx) for idx in range(start, end)])
+    start = int(self.cellsPerColumn * column)
+    end = int(start + self.cellsPerColumn)
```

Comment:
This change should actually just be:

    start = self.cellsPerColumn * self.getCellIndex(column)